src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -128,6 +227,24 @@ class AppleDeviceInterface extends InterfacePrototype {
         return wkSendAndWait('pmHandlerGetCreditCard', { id })
     }
 
+    // Used to encode data to send back to the child autofill
+    async selectedDetail (detailIn, configType) {
+        if (isTopFrame) {
+            let detailsEntries = Object.entries(detailIn).map(([key, value]) => {
+                return [key, String(value)]
+            })
+            const data = Object.fromEntries(detailsEntries)
+            wkSend('selectedDetail', { data, configType })
+        } else {
+            this.activeFormSelectedDetail(detailIn, configType)
+        }
+    }
+
+    async getCurrentInputType () {
+        const {inputType} = await wkSendAndWait('emailHandlerCheckAppSignedInStatus')
+        return inputType
+    }
```

Comment:
I don't get what this is supposed to do. This handler used to return a boolean, is that changed? Is it change only for macOS or for iOS as well?

Unless I'm misreading this, it seems like a hack we did at the beginning and now it's stuck there. If so and it's hard to cleanup now, please add a TODO.